komapper-annotation/src/main/kotlin/org/komapper/annotation/Annotations.kt

```diff
@@ -274,6 +274,30 @@ annotation class KomapperProjectionDef(
     val function: String = "",
 )
 
+/**
+ * The annotated class is a command that encapsulates an SQL template, SQL parameters, and SQL execution into a single unit.
+ *
+ * @property sql the SQL template
+ * @property disableValidation if `true`, SQL validation at compile time will be disabled
+ */
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.SOURCE)
+annotation class KomapperCommand(
+    val sql: String,
```

Comment:
Shouldn't there be `@Language("sql") ` on the sql property?

Reply:
Good point. I will fix it.